webserver/errors.py

```diff
@@ -1,8 +1,14 @@
-from flask import render_template
-
+from flask import render_template, jsonify
+import webserver.exceptions
 
 def init_error_handlers(app):
 
+    @app.errorhandler(webserver.exceptions.InvalidAPIUsage)
```

Comment:
Ah, right. No, I don't think it's needed (its still an error handler for the website)